applications/nrf_desktop/README.rst

```diff
@@ -8,15 +8,25 @@ Nordic Semiconductor HID reference design.
 Overview
 ********
 
-This application is a reference design of a HID device.
-
 The code is currently work-in-progress and as such is not fully functional, verified, or
 supported for product development.
 
+Welcome to the firmware documentation for nRF52 Desktop, a reference design of
+a HID device.
+
+Depending on a configuration this application can function as mouse,
+gaming mouse or keyboard connected to host via BLE and/or USB.
+
+Project supports common input HW interfaces i.e. motion sensors, rotation
+sensors and buttons scanning module.
+Firmware can be configured at run-time using dedicated configuration channel
+realized with HID feature report. The same channel is used to pass DFU packets.
+
+
 Requirements
 ************
 
-Supported boards:
+The project contains configuration files for following boards:
```

Comment:
Requirements and supported boards focus on two different things: not all boards mentioned in the list are required to use the firmware. Can we rephrase `Requirements` to `Supported boards`?

Reply:
I don't mind. I though that this is the Zephyr's convention. Samples and assert tracker mention boards in requirements.

Reply:
@b-gent , can you confirm this is a Zephyr convention?